### PR TITLE
Fix bug in UCR scheduled reports

### PR DESCRIPTION
**Pull request by NoahCarnahan**
The `ConfigurableReport.export_table()` method used the wrong column property to reference the returned data. This breaks emailed reports.

cc @gcapalbo 
fyi @nickpell

### CODE REVIEW

File: corehq/apps/userreports/tests/test_view.py [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+
+        view.request = MagicMock()
+        view.request.couch_user.language = "en"
+
+        view.report_config_id = self.report._id
+
+        self.assertEqual(
+            view.export_table,
+            [[
+                 u'foo',
```
Comment (dimagimon) [inline corehq/apps/userreports/tests/test_view.py]: E126 continuation line over-indented for hanging indent

File: corehq/apps/userreports/tests/test_view.py [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+                 [
+                     [u'report_column_display_fruit', u'report_column_display_percent'],
+                     [u'apple', '150%']
+                 ]
+            ]]
+        )
+
+
```
Comment (dimagimon) [inline corehq/apps/userreports/tests/test_view.py]: W391 blank line at end of file

File: corehq/apps/userreports/reports/specs.py [hunk 3]
```diff
@@ -119,7 +119,7 @@ def get_sql_column_config(self, data_source_config, lang):
                 header=self.get_header(lang),
                 agg_column=SQLAGG_COLUMN_MAP[self.aggregation](self.field, alias=self.column_id),
                 sortable=False,
-                data_slug=self.column_id,
+                slug=self.column_id,
```
Comment (NoahCarnahan) [inline corehq/apps/userreports/reports/specs.py]: This line was unrelated to the bug, but this keyword arg was wrong. I don't *think* that changing this line will change anything, but we should definitely wait for the tests.

**Comment by nickpell** [comment]
@NoahCarnahan crushing it

**Comment by nickpell** [comment]
test failure looks real

**Comment by NoahCarnahan** [comment]
@nickpell should be fixed

**Comment by nickpell** [comment]
looks good to merge when tests pass

**Comment by czue** [comment]
:heart: tests

**Pull request merged**